ext/standard/array.c

```diff
@@ -4964,7 +4899,7 @@ static void php_array_diff_key(INTERNAL_FUNCTION_PARAMETERS, int data_compare_ty
 	array_init(return_value);
 
 	/* Iterate over keys of the first array (handling possibility of indirects such as in $GLOBALS), to compute keys that aren't in the other arrays. */
```

Comment:
Should this and the above comment be updated? It seems like it no longer needs to handle this possibility and no longer does?